lib/generators/decidim/templates/Gemfile.erb

```diff
@@ -14,6 +14,9 @@ ruby "<%= RUBY_VERSION %>"
 
 gem "decidim", <%= gem_modifier %>
 
+# Uncomment the following line if you want to use decidim-assemblies plugin
+# gem "decidim-assemblies", <%= gem_modifier %>
```

Comment:
In my ideal world, features would be detachable as well, and generator would ask the user which participatory spaces and features she wants and add them to `Gemfile`. And inform her that they can always be added at a later stage to the Gemfile.

Well, in a super-ideal world, this would all be configurable via the admin or system UI through global settings. But that seems like dreaming at the moment :)